src/app/beer_garden/config.py

```diff
@@ -1165,10 +1214,12 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
     "amq": _AMQ_SPEC,
     "application": _APP_SPEC,
     "auth": _AUTH_SPEC,
+    #"routing": _ROUTING_SPEC,
     "configuration": _META_SPEC,
     "db": _DB_SPEC,
     "entry": _ENTRY_SPEC,
     "event": _EVENT_SPEC,
+    "garden_name": _GARDEN_NAME_SPEC,
```

Comment:
Can we reorganize this to:

```python
_GARDEN_SPEC = {
    "name": {
        ...
    }
}

...
"garden": _GARDEN_SPEC,
...
```

That way if we need to add anything to the garden part later we don't have to rearrange things.

Reply:
That makes sense. I changed it. Should we move the Namespace information to this location as well?